### ISSUE TITLE
About BatchNormalization in finetuning Stage

### ISSUE THREAD

**Issue by Kevinz-code** [hologerry/SoCo#13]
Hi, thanks for your great work and your code!

I notice that Sync BN was used in your models (e.g. SoCo_FPN_400ep), and there are SynBN operations in Backbone, FPN and RoI Head. So when I try to use your pretrained models like SoCo_FPN_400ep to do COCO-detection Task (say, faster_rcnn_fpn_res50), there could be three different choices about BN when finetuning on COCO-detection:

Possibly,
1. load all BN parameters (Backbone, FPN and RoI head) from SoCo_FPN_400ep,  fix all of them and use normal BN when finetuning on COCO
2. load all BN parameters (Backbone, FPN and RoI head) from SoCo_FPN_400ep, fix all of them and also use Sync BN when finetuning on COCO
3. load all BN parameters (Backbone, FPN and RoI head) from SoCo_FPN_400ep, set them as training parameters (do not fix) and use normal BN when finetuning on COCO

Could you please give me some suggestions?  many thanks in advance!

Best,
